hieroplatform/src/main/java/org/hiero/sketch/table/RecordOrder.java

```diff
@@ -179,4 +180,85 @@ public IndexComparator getComparator(@NonNull final Table table) {
         }
         return mergeLeft;
     }
+
+    /**
+     * Given two Tables in sorted order, decide the order in which to merge them. If two rows
+     * are equal, they will be combined to a single row (possibly with a larger  count).
```

Comment:
The counts are extraneous to this routine, which just flags the equality. It is up to whoever is calling it (like TopKSketch) to handle counts.